src/sss_client/idmap/sss_nss_idmap.c

```diff
@@ -198,8 +201,8 @@ static int  buf_to_kv_list(uint8_t *buf, size_t buf_len,
     return ret;
 }
 
-static int sss_nss_getyyybyxxx(union input inp, enum sss_cli_command cmd ,
-                               struct output *out)
+static int sss_nss_getyyybyxxx(union input inp, enum sss_cli_command cmd,
+                               int timeout, struct output *out)
```

Comment:
Is it expected that this parameter is type `int` but all the public functions define the timeout as `unsigned int` ?